galasa-ui/src/tests/components/test-runs/saved-queries/CollapsibleSideBar.test.tsx

```diff
@@ -40,6 +43,13 @@ jest.mock('@/contexts/TestRunsQueryParamsContext', () => ({
   useTestRunsQueryParams: jest.fn(),
 }));
 
+// Stylesheet seems to somehow override CollapsibleSideBar.module.css, so the extra necessary items have been added in here.
+jest.mock('@/styles/test-runs/TestRunsPage.module.css', () => ({
+  mainContent: 'TestRunsPage_mainContent__Ftan5',
+  sideNavCollapsed: 'sideNavCollapsed',
+  sideNavExpanded: 'sideNavExpanded',
+}));
+
```

Comment:
I don't think this is needed - removing this does cause the `should observe the main content if main content rendered, and set to height of main content -50px` test to fail, but setting the string 'mainContent' into the className seems to make the test pass again:

```ts
    test('should observe the main content if main content rendered, and set to height of main content -50px', async () => {
      const mainContentElement = document.createElement('div');
      mainContentElement.className = 'mainContent';
      document.body.appendChild(mainContentElement);
      ...
```